spec/normalize_country_spec.rb

```diff
@@ -108,6 +114,47 @@
     end
   end
 
+  describe ".extend_countries" do
+    let(:file_path) { File.join(File.dirname(__FILE__), "fixtures", "extra_countries", "en.yml") }
+    after { NormalizeCountry.reset! }
+
+    describe "when an array of hashes passed" do
```

Comment:
Can you use "with" instead of "when" to match [the existing specs that test method arguments](https://github.com/yuryroot/normalize_country/blob/b52b0316d9a336a874ad0d60e817c4b70452d4a3/spec/normalize_country_spec.rb#L98)? E.g., `"with an array of hashes"` no need for "passed".